src/components/home-page/HomeEndingSection.tsx

```diff
@@ -8,11 +8,12 @@ const HomeEndingSection = () => {
     <div className="bg-gradient-to-b from-transparent to-primary-wopee dark:to-secondary-wopee">
       <div className="container my-12 lg:my-16 h-[30vh] flex flex-col justify-center gap-5 lg:gap-10 text-center">
         <div className="text-balance px-2 lg:px-0">
-          <h2 className="lg:text-4xl sm:text-3xl text-xl font-bold">
-            Optimize your testing process with Wopee.io
+          <h2 className="lg:text-4xl sm:text-4xl text-4xl font-bold">
```

Comment:
If all screen sizes are using text-4xl  you can leave only one ```text-4xl``` and remove the rest (```sm:text-4xl``` and ```lg:text-4xl```)